Anna.Lytvyn/Lecture2/index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <link rel="shortcut icon" href="favicon.ico.png" type="image/x-icon">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.0/normalize.min.css">
+  <link rel="stylesheet" href="index.css" type="text/css">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat|Syncopate|PT+Mono" rel="stylesheet">
+  <title>Playground</title>
+</head>
+
+<body>
+  <!--Should be a fixed header-->
+  <header class="page-header">
+    <nav>
+      <a href="#"> Login </a>
+      <a href="#"> About us </a>
+      <a href="#"> Contact us </a>
+    </nav>
+  </header>
+  <figure class="top-image">
+    <!--Some 200px min-height image that takes all the horizontal space-->
+    <img src="figureimg.png">
```

Comment:
This image needs to be 100% width to spread across the screen on the wide screens. Now it sticks to the left. This is the perfect solution :)
```css
.jumbotron-image {
    width: 100%;
    height: 200px;
    background-position: center;
    object-fit: cover;
}
```